src/sentry/relay/projectconfig_debounce_cache/base.py

```diff
@@ -44,4 +44,6 @@ def mark_task_done(self, public_key, project_id, organization_id):
         """
         Mark a task done such that `check_is_debounced` starts emitting False
         for the given parameters.
+
+        Returns 1 if the task was removed, 0 if it wasn't.
```

Comment:
_nit_: We could use this as an occasion to introduce typing into projectconfig cache.

Reply:
Out of the scope of this PR, and I don't feel confident enough to add typing there and make sure it doesn't generate issues in places.